test/loopback.test.js

```diff
@@ -608,4 +611,96 @@ describe('loopback', function() {
       expect(methodNames).to.include('prototype.instanceMethod');
     });
   });
+
+  describe('Remote method inheritance', function() {
+    function getAllMethodNamesWithoutClassName(Model) {
+      return Model.sharedClass.methods().map(function(m) {
+        return m.stringName.replace(/^[^.]+\./, ''); // drop the class name
+      });
+    }
+    var app;
+
+    beforeEach(function() {
+      app = loopback({ localRegistry: true });
+    });
+
+    it('create model inherits method from base model', function() {
+      var Base = app.registry.createModel('Base', {}, {
+        methods: {
+          greet: {
+            http: { path: '/greet' },
+          },
+        },
+      });
+
+      var MyCustomModel = app.registry.createModel('MyCustomModel', {}, {
+        base: 'Base',
+        methods: {
+          hello: {
+            http: { path: '/hello' },
+          },
+        },
+      });
+      var methodNames = getAllMethodNamesWithoutClassName(MyCustomModel);
+
+      expect(methodNames).to.include('greet');
+      expect(methodNames).to.include('hello');
+    });
+
+    it('create and config model inherits method from base model', function() {
```

Comment:
I find the current test name not clear enough, it's not obvious what is important/specific in this particular test. I am proposing the following:

```js
it('inherits remote methods defined via configureModel')
```